src/graphics.c

```diff
@@ -1538,7 +1538,7 @@ GdipFillClosedCurve2 (GpGraphics *graphics, GpBrush *brush, GDIPCONST GpPointF *
 	if (tension == 0)
 		return GdipFillPolygon2 (graphics, brush, points, count);
 
-	if (!graphics || !brush || !points || count <= 0)
```

Comment:
Surely this changes behaviour if count==0

Reply:
@hughbe no because a few lines above we return from the function `if (count < 3)`